Packs/RedLock/Integrations/RedLock/RedLock.py

```diff
@@ -359,6 +362,28 @@ def translate_severity(alert):
     return 0
 
 
+def get_remediation_details():
+    """
+    Retrieve remediation details for a given alert
+    """
+    ids = argToList(demisto.getArg('alert-id'))
+    payload = {'alerts': ids, 'filter': {}}
+    handle_filters(payload['filter'])
+    handle_time_filter(payload['filter'], {'type': 'to_now', 'value': 'epoch'})
+    if not ids:
+        return_error('You must specify the alert-id to retrieve with CLI remediation')
+    response = req('POST', 'alert/remediation', payload, None)
+    MD = tableToMarkdown("remediationCLIoutput", response)
+    context = {'Redlock.Remediation': response}
```

Comment:
what is the unique key in `Remediation` object? If it is `alertIdVsCliScript` then update this line to be
`Redlock.Remediation(val. alertIdVsCliScript == obj.alertIdVsCliScript)`
https://xsoar.pan.dev/docs/integrations/context-and-outputs#dt-cortex-xsoar-transform-language

Reply:
@anara123  ahh the unique key should be the alert ID.  I made some modifications to handle this in a cleaner way which now results in `'Redlock.Alert(val.ID == obj.ID)'` being used. See the larger updates to the function for details.  New output format updated in docs and yml as well.  Thanks for your feedback!